### PR TITLE
Add AWS_SESSION_TOKEN to the documentation

### PR DESCRIPTION
**Pull request by 2opremio**
**What this PR does / why we need it**:

Improves documentation for users authentication with MFA

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:

None

**Special notes for your reviewer**:

None

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by vincepri** [comment]
@2opremio Thank you for the contribution! 🎉 

Overall LGTM, can we add some details to the PR using the repo's template?

**Comment by 2opremio** [comment]
@vincepri Done!

File: docs/getting-started.md [hunk 1]
```diff
@@ -64,8 +64,9 @@ administrator user you need to set your environment variables:
 * `AWS_REGION`
 * `AWS_ACCESS_KEY_ID`
 * `AWS_SECRET_ACCESS_KEY`
+* `AWS_SESSION_TOKEN` (if you are using Multi-factor authentication)
 
-After these are set run this command to get you up and running:
+- After these are set run this command to get you up and running:
```
Comment (vincepri) [inline docs/getting-started.md]: Is this extra?

Comment (2opremio) [reply docs/getting-started.md]: That was unintended, sorry. PTAL

**Pull request merged**